src/fortrancode.l

```diff
@@ -1101,7 +1162,19 @@ LANGUAGE_BIND_SPEC BIND{BS}"("{BS}C{BS}(,{BS}NAME{BS}"="{BS}"\""(.*)"\""{BS})?")
                                           else
                                           {
                                             g_insideBody=TRUE;
-                                            generateLink(*g_code, yytext);
+                                            // extract function name and process it
+                                            char *tmp= (char *) malloc(strlen(yytext)+1);
+                                            strcpy(tmp,yytext);
+                                            strtok(tmp," \t");
+                                            //printf("===> line number %d function call |%s|\n",g_yyLineNr,yytext);
+                                            generateLink(*g_code, tmp);
+                                            if ( strlen(tmp) != strlen(yytext) ) {
+                                              // everything else is thrown back into input stream
+                                              // to prevent {BS} from being gobbled
+                                              //printf("     reprocess |%s|\n",yytext+strlen(tmp));
+                                              yyless(strlen(tmp));
+                                            }
+                                            free(tmp);
```

Comment:
Why not move the `{BS}` after the `/` so no push back is necessary.

Reply:
On line 1165, {BS} after / would be nice, but generates "dangerous trailing context" compiler warning because {ID} contains a variable length wildcard (*).  Please suggest a simpler alternative.

Reply:
I was thinking about directly "codifying" the spaces and not pushing them back. To be able to give a good suggestion do you have a small example where this benefit shows?

Reply:
One example would be preserving the whitespace alignment of function names and args across multiple source lines, as in the main program below.

```
PROGRAM main
  n = fun1  (1)  ! call fun1
  n = fun100(1)  ! call fun100
CONTAINS
  FUNCTION fun1(n)
    fun1 =  abs(n)
  END FUNCTION fun1
  FUNCTION fun100(n)
    fun100 =  abs(n)*2
  END FUNCTION fun100
END PROGRAM main
```